datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/AddTermResolver.java

```diff
@@ -0,0 +1,60 @@
+package com.linkedin.datahub.graphql.resolvers.mutate;
+
+import com.linkedin.common.urn.CorpuserUrn;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.exception.AuthorizationException;
+import com.linkedin.datahub.graphql.generated.TermUpdateInput;
+import com.linkedin.metadata.entity.EntityService;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.concurrent.CompletableFuture;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.*;
+
+
+public class AddTermResolver implements DataFetcher<CompletableFuture<Boolean>> {
+  private static final Logger _logger = LoggerFactory.getLogger(MutableTypeResolver.class.getName());
```

Comment:
If you add the @Slf4j annotation, you can just use log without having to add this variable

Reply:
addressed!